### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by EricForever21**
Translation using poedit for Spanish language. I have created a branch called Patch-1 in which I have all the files under es_ES.

### CODE REVIEW

**Comment by pH-7** [comment]
Thanks a lot for your contribution Eric! Highly appreciate.

File: _protected/app/langs/es_ES/LC_MESSAGES/files [hunk 1]
```diff
@@ -0,0 +1,2 @@
+global.po
```
Comment (pH-7) [inline _protected/app/langs/es_ES/LC_MESSAGES/files]: Is there a reason you added this?

File: _protected/app/langs/es_ES/config/file [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (pH-7) [inline _protected/app/langs/es_ES/config/file]: Same for this one. This file can be deleted.

File: _protected/app/langs/es_ES/language.php [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+
+    /***** BYTE UNIT *****/
+    'byte' => '%0% Bytes',
+    'kilobyte' => '%0% KB',
+    'megabyte' => '%0% MB',
+    'gigabyte' => '%0% GB',
+    'terabyte' => '%0% TB',
+
+    /***** USER'S INFO FIELDS *****/
+    'address' => 'Address:',
```
Comment (pH-7) [inline _protected/app/langs/es_ES/language.php]: FYI, those strings are used in the custom user fields. If you want to have them in Spanish, they should be translated as well

File: _protected/app/langs/es_ES/language.php [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+    '+10.5' => '(UTC +10:30) Lord Howe Island',
+    '+11' => '(UTC +11:00) Magadan, Solomon Islands, New Caledonia',
+    '+11.5' => '(UTC +11:30) Norfolk Island',
+    '+12' => '(UTC +12:00) Auckland, Wellington, Fiji, Kamchatka',
+    '+12.75' => '(UTC +12:45) Chatham Islands',
+    '+13' => '(UTC +13:00) Phoenix Islands, Tonga',
+    '+14' => '(UTC +14:00) Line Islands',
+
+    /***** GENDER *****/
+    'male'=>'Man',
```
Comment (pH-7) [inline _protected/app/langs/es_ES/language.php]: For your information, those strings are used in the profile page. If you want to get them translated, it should be done as well

**Review by pH-7 — COMMENTED** [review]
A pH7CMS translation just needs the following files:

- _protected/app/langs/es_ES/config/config.ini

- _protected/app/langs/es_ES/LC_MESSAGES/global.mo

- _protected/app/langs/es_ES/LC_MESSAGES/global.po

- _protected/app/langs/es_ES/language.php


The other files can be deleted.

Same for `_protected/app/langs/es_ES/es_ES.html`. pH7CMS won't use it.

**Comment by EricForever21** [comment]
Great. I have completed per your reviews. Please let me know if everything
is ok now.

Best regards,
Eric

**Comment by pH-7** [comment]
It seems perfect by now @EricForever21 Muchas gracias!

**Review by pH-7 — APPROVED** [review]

**Comment by EricForever21** [comment]
Great! please let me know about the free license.
Best regards,
Eric

**Comment by pH-7** [comment]
Please send an email to hello@ph7cms.com with name, desire username and account email and I will create an account for you.

**Pull request merged**